packages/block-editor/src/components/text-transform-control/stories/index.story.js

```diff
@@ -1,33 +1,59 @@
-/**
- * WordPress dependencies
- */
-import { useState } from '@wordpress/element';
-
 /**
  * Internal dependencies
  */
 import TextTransformControl from '../';
 
-export default {
+const meta = {
 	title: 'BlockEditor/TextTransformControl',
 	component: TextTransformControl,
+	parameters: {
+		docs: {
+			canvas: { sourceState: 'shown' },
+			description: {
+				component:
+					'Control to facilitate text transformation selections.',
+			},
+		},
+	},
 	argTypes: {
-		onChange: { action: 'onChange' },
+		onChange: {
+			action: 'onChange',
+			control: {
+				type: null,
+			},
+			description: 'Handles change in text transform selection.',
+			table: {
+				type: {
+					summary: 'function',
+				},
+			},
+		},
+		className: {
+			control: { type: 'text' },
+			description: 'Class name to add to the control.',
+			table: {
+				type: { summary: 'string' },
+			},
+		},
+		value: {
+			options: [ 'none', 'uppercase', 'lowercase', 'capitalize' ],
+			control: { type: 'radio' },
+			description: 'Currently selected text transform.',
+			table: {
+				type: { summary: 'string' },
+				defaultValue: { summary: 'none' },
+			},
+		},
```

Comment:
> As with [the `TextAlignmentControl` component](https://github.com/WordPress/gutenberg/blob/6e8588865ed61c990227dfb9d1324a52f4ff39af/packages/block-editor/src/components/text-alignment-control/stories/index.story.js#L42-L49), it may be a good idea to make the `options` prop controllable by adding the `options` field.

Sorry, my suggestion above was incorrect.

Because, unlike `TextAlignmentControl`, this component does not allow us to change `options` as props.

I think the correct thing to do here is to display the `value` prop without the control:

```suggestion
		value: {
			control: { type: null },
			description: 'Currently selected text alignment value.',
			table: {
				type: { summary: 'string' },
			},
		},
```